Documentation/contributing/development/contributing_guide.rst

```diff
@@ -198,10 +278,20 @@ Pull request review process
    required to obtain the privileges to modify GitHub labels on the pull
    request.
 
+Dedicated expectation time for each member of Janitors team: Follow the next
+steps 1 to 2 times per day. Works best if done first thing in the working day.
+
+#. Review all PRs requesting for review in `for you <https://github.com/cilium/cilium/pulls?q=is%3Apr+is%3Aopen+draft%3Afalse+review-requested%3A%40me+sort%3Aupdated-asc>`_;
```

Comment:
These are all the PRs requesting reviews from me as a member of cilium/janitors, but also as a member of other teams or even personally. As far as I understand:

- It shows PRs that are not related to the janitors' duty.
- It does not show merge candidates (approved PRs with CI checks passing) if they do not request from janitors. We still have the `ready-to-merge` label, but it is not always used>

I wasn't able to find a good filter for any of those two items, is anyone using something more efficient?

Reply:
>It shows PRs that are not related to the janitors' duty.

Unfortunately there is not way to see which reviews are requested for a particular team

>It does not show merge candidates (approved PRs with CI checks passing) if they do not request from janitors. We still have the ready-to-merge label, but it is not always used>

this one will be commiter's fault, not janitor's

Reply:
I think we should advocate all Cilium contributors to use this reviews page s their point-of-contact for reviews rather than, eg, everyone iterating over the total list of PRs :-)

Re: `ready-to-merge` label I don't see any harm in adding a link here that says "check through the PRs with ready-to-merge and if they exist, consider merging them". (In the absence of a bot that will do this for us)

Reply:
Do we have a description of ready-to-merge label somewhere. I've never understood exactly when to set it and who sets it.

Reply:
https://github.com/cilium/cilium/pull/11174/files#r418665764